libs/feature/editor/src/lib/components/overview-upload/overview-upload.component.ts

```diff
@@ -20,7 +20,7 @@ import { UiInputsModule } from '@geonetwork-ui/ui/inputs'
 export class OverviewUploadComponent implements OnInit {
   @Input() metadataUuid: string
 
-  resourceFileName: string
+  imageAltText: string
```

Comment:
Is this really a good change? We are using the resource file name as the alt text, not the other way around.

Reply:
Yes... This is a point of view. My logic is to name the variable as it will be used in the code, not the source of it. If I have some number that I want to use as the the tax amout for exemple, then I will name my variable "taxAmount", not "someNumber".

Keeping the source as the name is disturbing for me.